src/samples/ChildBenefitsClaim/ConfirmationPage.tsx

```diff
@@ -152,7 +156,11 @@ const ConfirmationPage = ({ caseId }) => {
           <p className='govuk-body'> {t('WE_WILL_TELL_YOU_IN_14_DAYS')}</p>
           <p className='govuk-body'>
             <a
-              href='https://www.tax.service.gov.uk/feedback/ODXCHB'
+              href={
+                isUnAuth
+                  ? 'https://www.tax.service.gov.uk/feedback/ODXCHBUA'
+                  : 'https://www.tax.service.gov.uk/feedback/ODXCHB'
+              }
```

Comment:
create utils for this as this is repeated code.

Reply:
Instead of writing that to utils , I am creating and calling a function for this in same tsx file. Because only this confirmation.tsx file is having that code repeated.